### PR TITLE
Update _bertopic.py to address #2316

### LINKED ISSUE #2316: Can't get updated topic_embeddings_ when using pre-calculated document embeddings?
### Have you searched existing issues?  🔎

- [x] I have searched and found no existing issues

### Desribe the bug

I can use the `update_topics` function by passing in the appropriate documents and my manually generated topic labels, and using the auto-generated embeddings. This works as intended and I get the updated `topic_embeddings_`.
`topic_model.update_topics(docs, topics=topics)`

However, if I calculate the document embeddings myself and pass them to BERTopic like so:
`topic_model.fit_transform(docs, embeddings=embeddings)`
then the `topic_embeddings_` are not updated after using `update_topics`. This is because the call to `_create_topic_vectors` within `update_topics` is done without any arguments being passed, and would normally use `self.embedding_model`, which is None when I define my own embeddings. 

How should I perform the update to get new `topic_embeddings_` in this case? 
Could a kwarg for passing pre-calculated embeddings to `update_topics` be added?

### BERTopic Version

Version: 0.16.4

### PR DESCRIPTION
**Pull request by nhansendev**
Added "embeddings" kwarg to update_topics for use when pre-calculated document embeddings are used. Addresses issue #2316

# What does this PR do?

Fixes #2316

## Before submitting
- [ ] This PR fixes a typo or improves the docs (if yes, ignore all other checks!).
- [ ] Did you read the [contributor guideline](https://github.com/MaartenGr/BERTopic/blob/master/CONTRIBUTING.md)?
- [ ] Was this discussed/approved via a Github issue? Please add a link to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes (if applicable)?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by nhansendev** [comment]
The user could easily miss that they need to provide the embeddings to `update_topics`. Should there be some sort of warning?

**Comment by MaartenGr** [comment]
Might indeed be nice to have such a check here but I'm not sure whether it would cover all cases considering the `_create_topic_vectors` also has a few different options for creating topic vectors.

**Comment by MaartenGr** [comment]
I'm rerunning all jobs since the error didn't relate to your code but a connection issue with huggingface.
